app/concerns/exportable_as_csv.rb

```diff
@@ -17,9 +17,16 @@ def self.to_csv
         csv << exportable_attributes.values
 
         all.each do |item|
-          csv << exportable_attributes.keys.map { |attr| item.send(attr) }
+          csv << exportable_attributes.keys.map do |attr|
+            safe_value = prevent_csv_code_injection(item.send(attr))
+            safe_value
+          end
         end
       end
     end
+
+    def self.prevent_csv_code_injection(value)
+      value.to_s.starts_with?(/[\-+=@]/) ? value.gsub(/^([\-+=@].*)/, '.\1') : value
```

Comment:
```suggestion
      value.to_s.starts_with?(/[\-+=@]/) ? ".#{value}" : value
```

Aside from just being a little hard to read, either you don't need the `to_s` or you'll run into an issue with negative values:

```ruby
[2] pry(main)> value = -1
=> -1
[3] pry(main)> value.to_s.starts_with?(/[\-+=@]/) ? value.gsub(/^([\-+=@].*)/, '.\1') : value
NoMethodError: undefined method `gsub' for -1:Integer
from (pry):3:in `<main>'
```

And since `CSV.generate` doesn't appears to differentiate between numbers and strings, may as well just always return a string if you need to prepend the `.`.

Reply:
Good point. We do sometimes get nil values, which was the reasoning behind `to_s` - but of course, the `gsub` is redundant if we've already detected that it starts with a formula character.

Reply:
there _shouldn't_ be any negative numbers in this data, so I'm OK with them getting an extra '.' prefix if they do occur.